src/main/java/core/basesyntax/services/impl/ValidatorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.exceptions.LineParseException;
+import core.basesyntax.services.Validator;
+
+public class ValidatorImpl implements Validator {
+    private static final String VALID_LINE_PATTERN = "[a-z],\\w+,\\d+";
+
+    @Override
+    public void validate(String line) throws RuntimeException {
```

Comment:
do we need to write about RuntimeExceptions in method signature? remove it here and in the interface